Development-Guidelines/building-angular-project.md

```diff
@@ -14,7 +14,7 @@ The bottom line is the UI project has zero dependencies on asp.net or Windows. H
 Umbraco 7 needs a couple of things to run:
 
 ### Node.js 
-To compile and run the UI project you need Node.js installed, you can get that at [https://nodejs.org](nodejs.org) for both Windows and OSX.
+To compile and run the UI project you need Node.js installed, you can get that at [https://nodejs.org](https://nodejs.org) for both Windows and OSX.
```

Comment:
Could do something like this:
`To compile and run the UI project you need [Node.js](https://nodejs.org) installed, it is needed for both Windows and OSX.`

Compiles to:
To compile and run the UI project you need [Node.js](https://nodejs.org) installed, it is needed for both Windows and OSX.

Reply:
Looks good to me. 👌